### DISCUSSION TITLE
database snapshot in paid version ?

### DISCUSSION THREAD

**Discussion by Nico-L** [nhost/nhost#144] [Q&A] [answered]
Hello,

My projects are made for small structures with low needs in terms of ressources. So your free plan should fit perfectly these needs. But I experienced data loss (OVH premises fire a couple years ago) so I would like to have a way to recover the database in case of trouble.

Does the nhost paid plan offers database snapshot or backup?

thanks

**Comment by elitan** [comment, marked as answer]
Hi Nico, Yes this is the plan. We'll add backup support on the paid plan. Expect that in the coming weeks.

**Reply by csadai** [reply]
Will it be a standard backup that allows data portability?

**Reply by csadai** [reply]
Do you plan to expose a DB connection string too? Would be awesome

**Reply by elitan** [reply]
> Will it be a standard backup that allows data portability?

Yes.

> Do you plan to expose a DB connection string too? Would be awesome

Yes: https://github.com/nhost/nhost/issues/113

**Reply by csadai** [reply]
Terrific

**Reply by sangdth** [reply]
🕺🏻
